From b833aa320441ea1e03d279484e6c0269909cb640 Mon Sep 17 00:00:00 2001
From: Chen Gang G <gang.g.chen@intel.com>
Date: Wed, 13 Oct 2021 15:27:26 +0800
Subject: [PATCH] Kernelflinger: Disalbe UI display function

This can reduce 5 seconds boot delay in un-secure boot case

Tracked-On:
Signed-off-by: Chen Gang G <gang.g.chen@intel.com>
---
 kernelflinger.c | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/kernelflinger.c b/kernelflinger.c
index 5e57c57..3ecba2a 100644
--- a/kernelflinger.c
+++ b/kernelflinger.c
@@ -1112,8 +1112,7 @@ static VOID enter_fastboot_mode(UINT8 boot_state)
 
 	die();
 }
-
-static void bootloader_recover_mode(UINT8 boot_state)
+static void bootloader_recover_mode(UINT8 boot_state __unused)
 {
 	enum boot_target target;
 
@@ -1142,8 +1141,8 @@ static void bootloader_recover_mode(UINT8 boot_state)
 	die();
 }
 
-static VOID boot_error(enum ux_error_code error_code , UINT8 boot_state,
-			UINT8 *hash , UINTN hash_size )
+static VOID boot_error(enum ux_error_code error_code __unused, UINT8 boot_state,
+			UINT8 *hash __unused, UINTN hash_size __unused)
 {
 	BOOLEAN power_off = FALSE;
 	enum boot_target bt;
-- 
2.39.2

